gprofiler/merge.py

```diff
@@ -82,7 +82,85 @@ def collapse_stack(stack: str, comm: str) -> str:
     return ";".join(funcs)
 
 
-def parse_perf_script(script: str):
+def merge_global_perfs(
+    raw_fp_perf: Optional[str], raw_dwarf_perf: Optional[str]
+) -> Tuple[DefaultDict[int, MutableMapping[str, int]], Dict[int, str]]:
+    # The FP perf is used here as the "main" perf, to which the Dwarf perf is scaled.
+    merged_pid_to_stacks_counters: DefaultDict[int, MutableMapping[str, int]] = defaultdict(Counter)
+    fp_perf, fp_pid_to_name = parse_perf_script(raw_fp_perf)
+    dwarf_perf, dwarf_pid_to_name = parse_perf_script(raw_dwarf_perf)
+    dwarf_pid_to_name.update(fp_pid_to_name)
+    merged_pid_to_name = dwarf_pid_to_name
+
+    if raw_fp_perf is None:
+        return dwarf_perf, merged_pid_to_name
+    elif raw_dwarf_perf is None:
+        return fp_perf, merged_pid_to_name
+
+    total_fp_samples = sum([sum(stacks.values()) for stacks in fp_perf.values()])
+    total_dwarf_samples = sum([sum(stacks.values()) for stacks in dwarf_perf.values()])
+    fp_to_dwarf_total_sample_count_ratio = total_fp_samples / total_dwarf_samples
```

Comment:
Can you please log this number? `logger.debug`. It should remain roughly 1. If not, it can generate weird results. I wanna monitor it over the logs we have in our systems.